README.md

```diff
@@ -310,6 +310,9 @@ output_bags:
   compression_threads: 0
   include_hidden_topics: false
   include_unpublished_topics: false
+  # optional filter for msg time t [nsec since epoch]:  start_time_ns <= t <= end_time_ns
+  # start_time_ns: 1744227144744197147
+  # end_time_ns: 1744227145734665546
```

Comment:
i am not sure if we need to list all the possible options in the README, there are some other properties that are not listed here. besides, above description says `StorageOptions` and `RecordOptions`... i do not have any objection here.

but if we add them, probably we want to move these to right after `storage_config_uri` that is where `StorageOptions` belongs.

Reply:
moved documentation of start/end_time_ns to right after storage_config_uri.
I decided to submit this PR because slicing a bag by time (to reduce size) is a common operation and currently requires reading the source code to figure out how to do.
[Here](https://robotics.stackexchange.com/questions/97734/how-can-i-split-an-existing-bag-in-ros2) is the first time this came up on StackExchange.